### DISCUSSION TITLE
HAP Homebridge vs Homebridge Standalone

### DISCUSSION THREAD

**Discussion by demji1985** [thkl/hap-homematic#445] [General]
Hi, 

gibt es denn Unterschiede in der Art der Installation? Ich habe bei mir direkt mit dem Addon/Plugin begonnen, habe allerdings bisher schon vermehrt komische "Erfahrungen" gesammelt.

1.  Rollladen, der über Homematic Automation auf 40% gesetzt wird, erscheint in der Home App als "geschlossen" - geh ich auf den Status zeigt er es auch mit 0% an, nicht mit 40% - also keine Konfigurationssache. Eher ein Status-Problem...
2. Es dauert manchmal sehr lange, bis Geräte ihren Status aktualisieren, zwischenzeitlich bekomme ich auch mal "Keine Rückmeldung" oder "nicht erreichbar" bis es dann irgendwann wieder geht.

Erst dachte ich ja, es würde am iPad als Steuerzentrale liegen und besser werden, wenn das getauscht wird. Aber auch mit dem neuesten ATV jetzt, hat sich das nicht verbessert. Also muss das Problem irgendwo anders liegen.

Ich habe derzeit auch keine Firewall im Netz, die ggf. die Kommunikation stören könnte.

Hätte ich diese Probleme mit einer Docker App von Homebridge oder auch als VM genauso? Oder läuft das unter Umständen flüssiger?

**Comment by blackbasket** [comment]
Hi @demji1985 ,

zu 1: Ok, dann ist es wohl eher das Verhalten, dass ich bei mir auch beobachte. Für mich persönlich ist das low prio, da ich HomeKit nur zur Sprachsteuerung nutze ;) Ich habe das gleiche Verhalten mit meinen - über CuXD - angebundenen Somfy RTS-Rolles auch.

zu 2: Ist das ATV per Ethernet angebunden? Oder nutzt du die WLAN-NIC?

Zum HB-Plugin: Wenn es bei mir weiterhin mit HAP wöchentlich crasht, kann ich dir die Frage demnächst leider beantworten ;)

LG,
Marcel

**Reply by demji1985** [reply]
Das ATV ist aktuell noch per Wifi angebunden durch Mangel an Ports zur Fritzbox, steht aber quasi direkt daneben. 

Das klingt aber nicht gut ;-) Würdest du dann umsteigen auf Redmatic + Homebridge oder was wäre dann deine gewählte Lösung?
Da ich vorhab mir ein Nas mit Docker/VM zuzulegen, hätte ich dann etwas mehr Spielraum als nur das Raspmatic

**Reply by blackbasket** [reply]
Ok, mangels Port ist natürlich doof. Allerdings befürchte ich, dass das ATV bei WLAN ggf. auch mal einschläft.

Ich würde im ersten Step auf das HB-Plugin von Thomas umsteigen, da ich schon eine HB-VM laufen hab.
Es wäre erst einmal auch nur zum Test, um zu verifizieren, ob meine wöchentlichen Crashes tatsächlich von HAP kommen, oder nicht.

**Reply by demji1985** [reply]
Welches Plugin wäre das?

Kann man die parallel laufen lassen oder beißen die sich irgendwie?

Dann versuche ich mal die Ports umzumodeln und berichten, ob es besser wird.
